apply/templates/apply/dashboard_single_fund.html

```diff
@@ -53,6 +54,17 @@ <h2 class="govuk-heading-l">
                 {% if round["is_past_submission_deadline"] %}
                     {{ round_closed_warning(fund["fund_data"]["name"], round["round_details"]["title"], round["round_details"]["deadline"]) }}
                 {% endif %}
+                {% set ns = namespace(changes_requested_shown=False) %}
```

Comment:
I would try to avoid as much as possible setting variable in templates. Even more so updating them.

Appologies for my first suggestion, I tried it and it did not work. But looking at it a bit more, this should work:
`{% if 'CHANGES_REQUESTED' in round["applications"]|map(attribute='status') %}`

I think it's a more elegant solution.

Reply:
Nice, I would have never thought about doing it this way!